Code/MinimalLib/minilib.cpp

```diff
@@ -179,6 +248,96 @@ std::string JSMol::get_morgan_fp(unsigned int radius,
   return res;
 }
 
+std::string JSMol::get_stereo_tags() const {
+  if (!d_mol) return "{}";
+  rj::Document doc;
+  doc.SetObject();
+
+  bool cleanIt = true;
+  bool force = true;
+  MolOps::assignStereochemistry(*d_mol, cleanIt, force);
+
+  rj::Value rjAtoms(rj::kArrayType);
+  for (const auto atom : d_mol->atoms()) {
+    std::string cip;
+    if (atom->getPropIfPresent(common_properties::_CIPCode, cip)) {
+      cip = "(" + cip + ")";
```

Comment:
Out of curiosity, why are these surrounded by parentheses?

Reply:
That's what the current ketcher services do, apparently